src/utils/utils.ts

```diff
@@ -26,6 +26,10 @@ export function getServerUrl() {
     process.env.REACT_APP_SERVER_BASE_URL
 
 }
+export function getAuthServerUrl(){
```

Comment:
why function and not simply a variable which being initialized when app is loaded?